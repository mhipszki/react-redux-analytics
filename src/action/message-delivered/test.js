import sendMessageSuccess from './';

test('indicates message was sent successfully', () => {
  Date.now = jest.fn(() => 1234567890);
  expect(sendMessageSuccess()).toMatchSnapshot();
});
