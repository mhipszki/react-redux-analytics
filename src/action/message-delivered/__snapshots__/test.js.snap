// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`indicates message was sent successfully 1`] = `
Object {
  "timestamp": 1234567890,
  "type": "MESSAGE_DELIVERED",
}
`;
